scripts/data/src/org/akvo/gae/remoteapi/HidePrivateSurveyData.java

```diff
@@ -0,0 +1,148 @@
+/*
+ *  Copyright (C) 2015 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.gae.remoteapi;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+
+/**
+ * Check for survey data that is publicly visible and correct its privacyLevel
+ */
+public class HidePrivateSurveyData implements Process {
+
+    private static final int MAX_UNDERLYING_QUERIES = 99;
+
+    @Override
+    public void execute(DatastoreService ds, String[] args) throws Exception {
+
+        if (args.length == 0) {
+            System.err.println("Usage: " + RemoteAPI.class.getName()
+                    + "HidePrivateSurveyData <appid> <username> <password> <appid> [--dry-run]");
+            System.exit(1);
+        }
+        String appId = args[0];
```

Comment:
yeah I know its silly but thats because of the current setup of the RemoteAPI invocation. Let me see if there is a way to resolve it without messing up the other scripts